lib/vsc/install/shared_setup.py

```diff
@@ -1524,6 +1524,7 @@ def parse_target(self, target, urltemplate=None):
                     'lazy-object-proxy < 1.8.0', # higher requires python 3.7
                     'jsonpickle < 3.0.0', # higher requires python 3.7
                     'importlib-metadata < 5.0.0' # no longer compatible with python 3.7
+                    'dataclasses', # required by isort from pylint
```

Comment:
please add that it's part of python since 3.7 (I think?)

Reply:
That should be there already, since it's in the > 3 and < 3,7 branch.